app/src/main/java/dev/baseio/discordjetpackcompose/ui/routes/dashboard/friends/components/FriendRowComponent.kt

```diff
@@ -0,0 +1,179 @@
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import dev.baseio.discordjetpackcompose.R
+import dev.baseio.discordjetpackcompose.entities.ChatUserEntity
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import dev.baseio.discordjetpackcompose.ui.routes.dashboard.components.OnlineIndicator
+import dev.baseio.discordjetpackcompose.ui.theme.*
+
+@Composable
+fun FriendRowComponent(chatUserEntity: ChatUserEntity, isFriend: Boolean) {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(vertical = 5.dp),
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        ProfileComponent(chatUserEntity = chatUserEntity, isFriend)
+        if (isFriend) {
+            Row(
+                modifier = Modifier.wrapContentWidth(),
+                horizontalArrangement = Arrangement.End
+            ) {
+                ActionButton(
+                    icon = Icons.Default.Call,
+                    contentDescription = stringResource(id = R.string.call_friend)
+                )
+                Spacer(modifier = Modifier.width(30.dp))
+                ActionButton(
+                    painterResource =
+                    R.drawable.ic_chat_bubble,
+                    contentDescription = stringResource(id = R.string.message_friend)
+                )
+            }
+        } else {
+            Row(
+                modifier = Modifier.wrapContentWidth(),
+                horizontalArrangement = Arrangement.End
+            ) {
+                ActionButton(
+                    icon = Icons.Default.Close,
+                    tintColor = Color.Red,
+                    contentDescription = stringResource(id = R.string.remove_friend)
+                )
+                Spacer(modifier = Modifier.width(30.dp))
+                ActionButton(
+                    painterResource =
+                    R.drawable.ic_baseline_person_add_alt_1_24,
+                    contentDescription = stringResource(id = R.string.add_friend)
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun ActionButton(
+    background: Color = discord_icon_button_bg,
+    icon: ImageVector? = null,
+    painterResource: Int? = null,
+    contentDescription: String,
+    tintColor: Color = icon_button
+) {
+    IconButton(
+        onClick = { }, modifier = Modifier
+            .size(25.dp)
```

Comment:
This size is too big as compared to the original app. Let's fix this.

Reply:
always use dimens as multiple of 4,8,12,16,....